index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css" as="styles">
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+    @import url('https://fonts.cdnfonts.com/css/cocogoose');
+  </style>
+  <title>N'A.I'tion Evolution</title>
+</head>
+<body>
+  <div class="social-bar hidden">
+    <div class="container">
+      <ul class="social-bar__list flex">
+        <li class="social-bar__item"><a href="#"><img src="icons/facebook_icon.png" alt="Icon"></a></li>
+        <li class="social-bar__item"><a href="#"><img src="icons/twitter_icon.png" alt="Icon"></a></li>
+        <li class="social-bar__item"><a href="#">English</a></li>
+        <li class="social-bar__item"><a href="#">My Page</a></li>
+        <li class="social-bar__item"><a href="#">Logout</a></li>
+      </ul>
+    </div>
+  </div>
+
+  <header id="header" class="header p-fixed hidden">
+    <div class="header__logo"><img src="images/naition_logo.jpg" alt="Logo"></div>
+    <nav class="nav-bar">
+      <ul class="nav-bar__list">
+        <div class="nav-bar__item nav-bar__item--button"><button class="nav-bar__button"><img src="icons/nav_bar_close.png" alt="Close"></button></div>
+        <div><hr><li class="nav-bar__item"><a href="/index.html">Home</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="/about.html">About</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">Program</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">Join</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">Sponsors</a></li></div>
+        <div><hr><li class="nav-bar__item"><a href="#">News</a></li></div>
+        <div><hr><li class="nav-bar__item"><a class="nav-bar__item--design" href="#">NE Campaign</a></li><hr></div>
+      </ul>
```

Comment:
- [ ] Hello @lRebornsl, I notice i could not navigate to the `about page` from the `home page` and anytime i clicked on the about link it throw an error 404. I would kindly suggest you target the page directory very well and be sure its working.
possible implementation 👍 

  - I would suggest you to remove the slash at the back of the targeted directory.
  
```i.e 
<a href="index.html">Home</a>
<a href="about.html">Home</a>
```
Hopefully it will work.


Below is a video to my claim 👇 


https://github.com/lRebornsl/capstone-project/assets/78634841/5dcd2e1e-5405-46fe-a0b6-e9f0303a39b9